source/documentation/standards/hosting.md.erb

```diff
@@ -0,0 +1,66 @@
+---
+title: How to host services
+last_reviewed_on: 2020-09-01
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+At MOJ you should use the [Cloud
+Platform](https://user-guide.cloud-platform.service.justice.gov.uk/) to
+host your service unless you're advised otherwise by a member of [the
+hosting team](https://peoplefinder.service.gov.uk/teams/hosting). Your
+team can get started on the Cloud Platform without letting us know;
+just follow [the Cloud Platform documentation
+](https://user-guide.cloud-platform.service.justice.gov.uk/) and get in
+touch with the team if you need help.
+
+If you think that your service might require other hosting, [get in
+touch with the hosting
+team](mailto:hosting-leads@digital.justice.gov.uk) and we'll help
+identify which of our other hosting options is best for your service,
+or set up extra accounts if we've already identified the right
+environment for you:
+
+- AWS Modernisation Platform (currently in alpha)
+- Self-managed AWS infrastructure
+- LAA AWS and 6Degrees infrastructure
+- HMPPS Azure infrastructure
+- Retirement infrastructure
+
+## We use public cloud hosting by default
+
+We follow the [Government Cloud First
+policy](https://www.gov.uk/guidance/government-cloud-first-policy) and
+use public cloud Platform as a Service (PaaS) and Infrastructure as a
+Service (IaaS) infrastructure to host our services.
+
+Our choice of cloud platforms:
+
+- are highly scalable and available to meet the needs of service users
+- have automated tools for MOJ administrators to manage their
+  environments
+
+We currently use [Amazon Web Services (AWS)](https://aws.amazon.com/)
+by default for scalable computing, storage and deployment services, but
+we may recommend some services are hosted in [Microsoft
+Azure](https://azure.microsoft.com/), based on the needs of the service
+and its relationship to other services already in Azure.
+
+If your service is currently hosted outside of AWS or Azure, get in
+touch with [the Application Modernisation team](#TODO) to plan for its
+future hosting.
+
+## Paying for hosting
```

Comment:
This doesn't fit the tense of the other heading; it should be `We pay for hosting centrally by default`